Classes/Plugin/Calendar.php

```diff
@@ -297,7 +299,7 @@ protected function getCalendarYear($calendarIssuesByMonth, $year, $firstMonth =
                                         $dayLinkLabel = empty($issue['title']) ? strftime('%x', $currentDayTime) : $issue['title'];
                                         $linkConf = [
                                             'useCacheHash' => 1,
-                                            'parameter' => $this->conf['targetPid'],
+                                            'parameter' => $this->conf['linkToPurl'] ? $issue['purl'] : $this->conf['targetPid'],
```

Comment:
Before linking to `$issue['purl']` we should check if it isn't empty. Otherwise we have to fall back to using the internal linking.

```suggestion
                                            'parameter' => $this->conf['linkToPurl'] && !empty($issue['purl']) ? $issue['purl'] : $this->conf['targetPid'],
```

Also, when using the PURL the additional parameters are obsolete and should be omitted.